README.md

```diff
@@ -1 +1,15 @@
-# image-lambda
\ No newline at end of file
+# Lab 17: S3 and Lambda
+
+* Create a 50x50 pixel thumbnail version of that image
+* Save it to another S3 bucket.
+* It should do so with a predictable naming convention, so that your server and/or frontend know where that thumbnail image will be.
+
+## Author: Jason Dormier
+
+### Link to GitHub [repository](https://github.com/JasonDormier/image-lambda)
+
+### Using lambda create thumbnail function
+* drop an image in the buckiebucket bucket.
+* run invoke the createThumbnail function 
+* check the buckiebucket-resize for the resized image.
+
```

Comment:
In your README could you please link the tutorial that you used for this please!